vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -174,11 +174,11 @@ Another way to group together pages in the navigation is to use a [`NavBar`][viz
     [NavBar]: ../../assets/user_guides/navigation/nav_bar.png
 
 
-Here, the first level of the navigation hierarchy ("Group A" and "Group B") is represented by an icon in a navigation bar, and the second level of the navigation (the pages) is represented by an accordion. By default, the icons are the [`filter` icons from the Google Material icons library](https://fonts.google.com/icons?icon.query=filter).
+Here, the first level of the navigation hierarchy ("Group A" and "Group B") is represented by an icon in a navigation bar, and the second level of the navigation (the pages) is represented by an accordion. By default, the icons are from [the Google Material `filter` icons library](https://fonts.google.com/icons?icon.query=filter).
```

Comment:
This made more sense to me before because the Google Material icons library is the library as a whole, and the `filter` icons are just some icons inside that library?

Maybe this would be clearer than the original?
```
By default, the set of icons used are the [`filter` icons from the Google Material icons library](https://fonts.google.com/icons?icon.query=filter).
```

Reply:
Yeah, that makes more sense! I think the filter notation is just very confusing if one hasn't understood that this is just a category name for a set of Google icons. Will change to above :)